apps/meteor/app/livechat/imports/server/rest/appearance.ts

```diff
@@ -52,8 +52,39 @@ API.v1.addRoute(
 				throw new Error('invalid-setting');
 			}
 
+			const dbSettings = await Settings.find({ _id: { $in: validSettingList } }, { projection: { _id: 1, value: 1, type: 1 } }).toArray();
+
+			const settingsToSave: typeof this.bodyParams = [];
+			dbSettings.forEach((dbSetting) => {
+				const setting = settings.find(({ _id }) => _id === dbSetting._id);
+				if (!setting || dbSetting.value === setting.value) {
+					return;
+				}
+
+				switch (dbSetting?.type) {
+					case 'boolean':
+						settingsToSave.push({
+							_id: dbSetting._id,
+							value: coerceValue(setting?.value, 'boolean'),
```

Comment:
At this point, we already know the data type, Is there any reason to create the function to coerce multiple data types, instead of comparing it here directly? e.g `(value === 'true' || value === true)`

Reply:
idk, I just liked the function. Initially this copied the multiple data types the settings can have, but then I realized i only needed string|bool|int